src/modelio.cs

```diff
@@ -2946,6 +2947,16 @@ interface MDLTransformMatrixOp : MDLTransformOp {
 		MDLAnimatedMatrix4x4 AnimatedValue { get; }
 	}
 
+	[iOS (12,0), Mac (10,14), TV (12,0)]
+	[BaseType (typeof (NSObject))]
+	interface MDLTransformOrientOp : IMDLTransformOp {
```

Comment:
the type is marked as 12.0
https://developer.apple.com/documentation/modelio/mdltransformorientop?language=objc
but all members are 13.0
https://developer.apple.com/documentation/modelio/mdltransformorientop/2890059-animatedvalue?language=objc
https://developer.apple.com/documentation/modelio/mdltransformorientop/2889963-name?language=objc

my guess is that the former was an error - but, in any case, the member are now incorrectly attributed